hudi-spark-datasource/hudi-spark/src/main/scala/org/apache/spark/sql/hudi/command/procedures/RunCompactionProcedure.scala

```diff
@@ -147,7 +147,13 @@ class RunCompactionProcedure extends BaseProcedure with ProcedureBuilder with Sp
   private def handleResponse(metadata: HoodieCommitMetadata): Unit = {
     // Handle error
     val writeStats = metadata.getPartitionToWriteStats.entrySet().flatMap(e => e.getValue).toList
-    val errorsCount = writeStats.map(state => state.getTotalWriteErrors).sum
+    val errorsCount = writeStats.map { state =>
+      val writeErrors = state.getTotalWriteErrors
+      if (writeErrors > 0) {
+        logError(s"Error occurred while writing the file: ${state.getPath}.")
+      }
```

Comment:
Do we need to log each file?

Reply:
No, we only print those files which `getTotalWriteErrors > 0`

Reply:
How about we got too many error files?

Reply:
Emm, limit 100  error files?

Reply:
Yeah, guess we need some protection to avoid deluge of file path loggings.

Reply:
Done